tests/integration_tests/test_botorch.py

```diff
@@ -84,6 +84,60 @@ def candidates_func(
     assert candidates_func_call_count == n_trials - n_startup_trials
 
 
+@pytest.mark.parametrize("candidates_func_kind", ["qei", "qehvi", "qparego", "qnhevi"])
```

Comment:
It would be great that the branching only appears in the parametrize. How about the following?
```suggestion
@pytest.mark.parametrize("candidates_func, n_objectives", [
    (integration.botorch.qei_candidates_func, 1),
    (integration.botorch.qehvi_candidates_func, 2),
    (integration.botorch.qparego_candidates_func, 4),
    (integration.botorch.qnehvi_candidates_func, 2),
])
```

Reply:
Thank you for the better code. I adopted the suggestion.
Additionally, I added `(integration.botorch.qnehvi_candidates_func, 3)` to cover the case `alpha > 0`.